src/test/regress/sql/incremental_matview.sql

```diff
@@ -185,6 +185,25 @@ DELETE FROM mv_base_a;
 SELECT * FROM mv_ivm_min_max;
 ROLLBACK;
 
+-- aggregate views with column names specified
+BEGIN;
+CREATE INCREMENTAL MATERIALIZED VIEW mv_ivm_agg(a) AS SELECT i, SUM(j) FROM mv_base_a GROUP BY i;
+INSERT INTO mv_base_a VALUES (1,100), (2,200), (3,300);
+UPDATE mv_base_a SET j = 2000 WHERE (i,j) = (2,20);
+DELETE FROM mv_base_a WHERE (i,j) = (3,30);
+SELECT * FROM mv_ivm_agg ORDER BY 1,2;
+ROLLBACK;
+BEGIN;
+CREATE INCREMENTAL MATERIALIZED VIEW mv_ivm_agg(a,b) AS SELECT i, SUM(j) FROM mv_base_a GROUP BY i;
```

Comment:
This test case is repeated.

Reply:
No, these tests are different. Note the view definitions:

`CREATE INCREMENTAL MATERIALIZED VIEW mv_ivm_agg(a)`
vs
`CREATE INCREMENTAL MATERIALIZED VIEW mv_ivm_agg(a,b)`

Reply:
sorry, that's ok